grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -130,6 +130,13 @@ class NewConsentGroupController extends AuthenticatedController {
         render([message: 'Consent Group was updated'] as JSON)
     }
 
+    def deleteNoConsentReason() {
+        Issue issue = Issue.findByProjectKey(params.consentKey)
```

Comment:
Avoid using GORM

Reply:
👍 to this suggestion. The conversation is marked as resolved, but the suggestion was not implemented.